endpoints.json

```diff
@@ -23,5 +23,77 @@
         }
       ]
     }
+  },
+
+  "GET /api/articles/:article_id": {
+    "description": "serves information about a specific article",
+    "queries": ["article_id"],
+    "exampleResponse": {
+      "article_id": 5,
+      "title": "UNCOVERED: catspiracy to bring down democracy",
+      "topic": "cats",
+      "author": "rogersop",
+      "body": "Bastet walks amongst us, and the cats are taking arms!",
+      "created_at": "2020-08-03T13:14:00.000Z",
+      "votes": 0,
+      "comment_count": 2
+    }
+  },
+
+  "PATCH /api/articles/:article_id": {
+    "description": "takes an object for updating information and serves the updated article",
+    "queries": [{ "inc_votes": "numVotes" }],
+    "exampleRequest": {
+      "inc_votes": 1
+    },
+    "exampleResponse": {
+      "article_id": 4,
+      "title": "Student SUES Mitch!",
+      "topic": "mitch",
+      "author": "rogersop",
+      "body": "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+      "created_at": "2020-05-06T01:14:00.000Z",
+      "votes": 1
+    }
+  },
+
+  "GET /api/users": {
+    "description": "serves users and their username, name and avatar photo",
+    "queries": [],
+    "exampleResponse": {
+      "username": "butter_bridge",
+      "name": "jonny",
+      "avatar_url": "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg"
+    }
+  },
+
+  "GET /api/articles/:article_id/comments": {
+    "description": "serves all comments from a specific article and the comment information",
+    "queries": [],
+    "exampleResponse": {
+      "comment_id": 2,
+      "body": "The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.",
+      "article_id": 1,
+      "author": "butter_bridge",
+      "votes": 14,
+      "created_at": "2020-10-31T03:03:00.000Z"
+    }
+  },
+
+  "POST /api/article/:article_id/comments": {
+    "description": "takes a new comment and adds it to the database then serves the newly posted comment",
+    "queries": [],
+    "exampleRequest": {
+      "username": "lurker",
+      "body": "As you were x"
+    },
+    "exampleResponse": {
+      "comment_id": 19,
+      "body": "As you were x",
+      "article_id": 2,
+      "author": "lurker",
+      "votes": 0,
+      "created_at": "2022-02-17T20:12:37.478Z"
+    }
```

Comment:
Just remember to keep updating this as you make more endpoints - it will be really useful when you get to front end.